development/tools/jobs/kyma/kyma_integration_test.go

```diff
@@ -318,4 +318,21 @@ func TestKymaIntegrationJobPeriodics(t *testing.T) {
 	tester.AssertThatContainerHasEnv(t, backupRestorePeriodic.Spec.Containers[0], "REPO_OWNER_GIT", "kyma-project")
 	tester.AssertThatContainerHasEnv(t, backupRestorePeriodic.Spec.Containers[0], "REPO_NAME_GIT", "kyma")
 
+	expName = "kyma-load-tests-weekly"
+	loadTestPeriodic := tester.FindPeriodicJobByName(periodics, expName)
+	require.NotNil(t, loadTestPeriodic)
+	assert.Equal(t, expName, loadTestPeriodic.Name)
+	assert.True(t, loadTestPeriodic.Decorate)
+	assert.Equal(t, "0 2 * * 1", loadTestPeriodic.Cron)
+	tester.AssertThatHasPresets(t, loadTestPeriodic.JobBase, tester.PresetGCProjectEnv, tester.PresetSaGKEKymaIntegration, "preset-load-test-slack-notifications", "preset-weekly-github-integration")
+	tester.AssertThatHasExtraRefs(t, loadTestPeriodic.JobBase.UtilityConfig, []string{"test-infra", "kyma"})
+	assert.Equal(t, "eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20190129-c951cf2", loadTestPeriodic.Spec.Containers[0].Image)
+	assert.Equal(t, []string{"bash"}, loadTestPeriodic.Spec.Containers[0].Command)
+	assert.Equal(t, []string{"-c", "${KYMA_PROJECT_DIR}/test-infra/prow/scripts/cluster-integration/kyma-gke-load-test.sh"}, loadTestPeriodic.Spec.Containers[0].Args)
+	tester.AssertThatSpecifiesResourceRequests(t, loadTestPeriodic.JobBase)
+	assert.Len(t, loadTestPeriodic.Spec.Containers[0].Env, 6)
```

Comment:
you have an assertion that checks if  there are 6 envs and then you check the value of only 4